connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/functions.scala

```diff
@@ -80,14 +81,37 @@ object functions {
    *
    * @param data
    *   the binary column.
-   * @param shadedMessageClassName
+   * @param messageClassName
    *   The Protobuf class name. E.g. <code>org.spark.examples.protobuf.ExampleEvent</code>.
    *   The jar with these classes needs to be shaded as described above.
    * @since 3.4.0
    */
   @Experimental
-  def from_protobuf(data: Column, shadedMessageClassName: String): Column = {
-    new Column(ProtobufDataToCatalyst(data.expr, shadedMessageClassName))
+  def from_protobuf(data: Column, messageClassName: String): Column = {
+    new Column(ProtobufDataToCatalyst(data.expr, messageClassName))
+  }
+
+  /**
+   * Converts a binary column of Protobuf format into its corresponding catalyst value. The
+   * specified Protobuf class must match the data, otherwise the behavior is
+   * undefined: it may fail or return arbitrary result. The jar containing Java class should be
+   * shaded. Specifically, `com.google.protobuf.*` should be shaded to
```

Comment:
to match with the spark-doc, should we also update this and include a reference to the shading example?

Reply:
Do you mean linking to https://github.com/rangadi/shaded-protobuf-classes ?

Reply:
Yes

Reply:
@SandishKumarHN made a big update to both Scala and Python functions. PTAL.